docs/visual-testing/faq.md

```diff
@@ -0,0 +1,39 @@
+---
+id: faq
+title: Sauce Visual Testing FAQ
+sidebar_label: FAQ
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+### Is Sauce Visual's image comparison pixel-based or DOM-based?
+
+The current implementation is Pixel-based only.
+
+### Does Sauce Visual support "ignore regions"?
+
+Yes, "ignore regions" are supported. Currently, you can define these regions in code by specifying the locator of a Web Element or by setting the coordinates of a rectangle.
+
+```java
+  // This rectangular area would be ignored, no  pixel
+  // changes will be diffed in this zone.
+  IgnoreRegion ignoreRegion = new IgnoreRegion(200,200,100,100);
+```
+
+### Can I integrate this into CI/CD testing?
+
+Yes, Sauce Visual is 100% API-based and natively utilizes the same tools as the rest of the Sauce product line. When integrating Visual testing into your scripts, you can execute them just like any other scripts.
+
+### What languages/frameworks will be supported at launch?
```

Comment:
```suggestion
## What languages/frameworks will be supported at launch?
```

Reply:
We set it to h3 because otherwise the question would become to big